vite/msdfGenerator.js

```diff
@@ -158,6 +163,44 @@ const copyDir = (src, dest) => {
     const srcPath = path.join(src, entry.name)
     const destPath = path.join(dest, entry.name)
 
+    // Skip copying charset.txt file into dist directory
+    if (srcPath.includes('.charset.txt')) continue
+
     entry.isDirectory() ? copyDir(srcPath, destPath) : fs.copyFileSync(srcPath, destPath) // Copy files
   }
 }
+
+const createCharsetFile = (charsetDir, ConfigDir, fontName) => {
+  const charsetFilePath = path.join(charsetDir, `${fontName}.charset.txt`)
+
+  // Check charset file already exists, if exists, return file path
+  if (fs.existsSync(charsetFilePath)) {
```

Comment:
This check means that making changes to the charset config will never be reflected

Reply:
Before that we have one more check, ```msdf.json``` or ```.msdf.png```  available in ```.tmp-msdf-fonts-v2``` directory or not.  if not exist then only we are going to generate them.

I think, User must delete ```.tmp-msdf-fonts-v2``` directory to generate again.

Reply:
okay, yeah that's an interesting point - cause I can imagine a user adding something to the config later on. that should ideally lead to an automatic regeneration

Reply:
I had an idea for solving this: instead of looking if the file already exists in `.tmp-msdf-fonts-v2`, can't we compare file dates? If the font file or the config file (in the `assets` folder) are newer than those generated in the `.tmp-msdf-fonts-v2`-folder, something must have changed so we generate again

This removes the need for a user to ever manually delete `.tmp-msdf-fonts-v2` to generate again ...

Does this make sense?

cc @uguraslan

Reply:
I suggest we save the checksum of any config files associated with each font into the `tmp-msdf-fonts-v2` directory. For each font file, the process would be:

- Check if the font has an associated config file.
- If a config exists, check if a previously calculated checksum is saved.
  - If no checksum is found, regenerate the font and save the new checksum.
  - If a checksum is found, compare it with the current checksum. If they differ, regenerate the font and update the checksum.
- If no config file exists, check if the font has already been generated.
  - If it hasn't been generated, create it.
  - If it has, do nothing.
  
For the checksum calculation, we can use a fast algorithm like `md5`, as we don't need cryptographic security—just a basic change detection. The checksums could be stored in files like `.tmp-msdf-fonts-v2/font-name.msdf.checksum` for each font with a config file.

Reply:
As per your suggestion @uguraslan , we are going to generate checksum for config.json not for .ttf file.  As ```font.config.json``` would be small size, I feel, generating checksum everytime in ```dev``` mode does not alter performance that can be noticed.  

There are very less chances that the ```.ttf``` file get modified but still, I feel, we should address that as well but creating checksum for ```ttf``` file could be more intensive task. 

I feel, reading file metadata is less costly here for ```ttf``` files. like modified time